jax/experimental/jax2tf/jax2tf.py

```diff
@@ -675,6 +675,107 @@ def _conv_general_precision_config_proto(precision):
   proto.operand_precision.append(int(precision))
   return proto
 
+def _can_use_tf_conv(lhs, rhs, window_strides, padding,
```

Comment:
Thinking ahead towards the use case classifier, I think it would be useful for this function to return also a string with the reason why one cannot use tf conv. The classifier can use that string to classify what are the common obstacles.

Reply:
Very good idea, and done in the latest version!